functions/index.js

```diff
@@ -0,0 +1,153 @@
+const axios = require('axios');
+const btoa = require('btoa')
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+
+admin.initializeApp({
+    credential: admin.credential.applicationDefault(),
+    databaseURL: "https://tuneiq.firebaseio.com"
+});
+
+exports.createGame = functions.https.onRequest((request, response) => {
+    const gameGenre = request.query.genre
+    const hostId = request.query.hostId
+    const gameName = request.query.name
+    const password = request.query.password
+    const gameRounds = functions.config().gameconfig.number_of_rounds
+
+    getRoundDetails(gameGenre, gameRounds).then(roundDetails => {
+        gameDetails = {
+            hostId: hostId,
+            name: gameName,
+            password: password,
+            genre: gameGenre,
+            roundDetails: roundDetails,
+            players: [hostId]
+        }
+        admin.firestore().collection("games").add(gameDetails).then(entry => {
```

Comment:
i dont think this cloud function should be creating the game. the frontend app creates the game in the db after the user enters details, and then on success passes those params here and this should be concerened with only creating the game play aspect (i.e. not adding to the firestore db). open for other thoughts

Reply:
Going to leave the db writing for now. We can change it later

Reply:
ok sounds good. it looks like we have 2 dbs in the firestore console, can you double check that you dont need the realtime db? we should be using the firestore db instead of the realtime db

Reply:
yeah it uses firestore now